SystemLog/Logger.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.IO;
+
+namespace Neo.Plugins
+{
+    public class Logger : Plugin, ILogPlugin
+    {
+        public override string Name => "SystemLogs";
+
+        public override void Configure()
+        {
+            Settings.Load(GetConfiguration());
+        }
+
+        public void Log(string source, LogLevel level, string message)
+        {
+            var log = $"[{DateTime.Now.TimeOfDay:hh\\:mm\\:ss\\.fff}] {message}";
+
+            if (Settings.Default.ConsoleOutput)
+            {
+                var currentColor = new ConsoleColorSet();
+
+                switch (level)
```

Comment:
There is no other way, right @shargon? It is strange to set this color all the time.
With this we could also debug different classes with different levels, yes? Because a cache for each part of the code could be more efficiently, however...if we are printing efficiency is not a focus anyway.

A major configuration could easily switch the logs for different parts of the codes.